.buildkite/release-pipelines/finalize-release.yml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+steps:
+  - label: Finalize Release
+    plugins: [$CI_TOOLKIT_PLUGIN]
+    command: |
+      echo '--- :robot_face: Use bot for git operations'
+      source use-bot-for-git wpmobilebot
+
+      echo '--- :git: Checkout Release Branch'
+      .buildkite/commands/checkout-release-branch.sh
```

Comment:
I plan to followup with a dedicated PR that surfaces `RELEASE_VERSION` here and remove reading it from the environment in the script.

I think by doing that we might also make the script more portable and possibly move it to the CI plugin. Being such a simple script, I wonder what's the value, but then again, it would make the implementation, the error handling, and the messages it print consistent across project, which I think is desirable for us.